lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,26 @@
+<div id="change-session-<%-entitlementUUID%>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('To change your session or leave your current session, please select from the following:')%>
+        <% } else { %>
+            <%- gettext('In order to view the course you must select a session:')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select">
+            <% for (var i=0; i< availableSessions.length; i++) { %>
```

Comment:
It's more common to use ```_.each``` loops in templates (http://underscorejs.org/#each).

Here's an example https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/cms/templates/js/settings-modal-tabs.underscore#L2-L6

Reply:
Changed